### PR TITLE
CB-5 Login for company

### PR DESCRIPTION
**Pull request by Ry4n6137**

### CODE REVIEW

**Comment by Qiming-Liu** [comment]
What's the router url for this page?

File: src/pages/Login/components/LoginForCompany/FormBuilder/FormBuilder.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react'
+import file from '../../../assert/file.svg'
+
+const FormBuilder: FC = () => (
+  <div className=" flex justify-center items-center bg-[#03111B] h-[832px] w-[640px] ">
+    <div className="text-center">
+      <div className="w-[77px] h-[99px] mx-auto">
+        <img className="w-[77px] h-[99px] " src={file} />
+      </div>
+      <div className="text-[#FFFFFF] w-[131px] h-[27px] text-[22px] font-medium leading-[22.63px]">
```
Comment (smarthulala) [inline src/pages/Login/components/LoginForCompany/FormBuilder/FormBuilder.tsx:10]: #FFFFFF equals white
leading-[22.63px] is this necessary? use 22px or w-6

File: src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import { FC } from 'react'
+
+const LoginPage: FC = () => (
+  <div className="flex justify-center items-center bg-[#FFFFFF] w-[640px] h-[832px]  ">
+    <div className="relative text-left w-[280px] h-[356px]">
+      <h1 className="text-[#1E1E1E] text-[22px] mb-5 font-medium">Welcome back, Alonso</h1>
+      <div>
+        <div>
+          <p className="text-[16px] leading-[19px] font-medium not-italic">Work email</p>
```
Comment (smarthulala) [inline src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx:9]:
```suggestion
          <p className="text-4 leading-5 font-medium not-italic">Work email</p>
```

File: src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          <input
+            className="box-border pl-[3px] w-[280px] h-[44px] mb-[24px] border-solid border-[1px] border-[#03111B] rounded-[5px]
```
Comment (smarthulala) [inline src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx:11]:
```suggestion
            className="box-border pl-[3px] w-[280px] h-11 mb-6 border border-[#03111B] rounded-[5px]
```

File: src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+             placeholder-shown:border-gray-500"
+            placeholder="  Alonso@mail.com"
+          />
+        </div>
+        <div>
+          <p className="text-[16px] leading-[19px] font-medium not-italic">Your site</p>
```
Comment (smarthulala) [inline src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx:17]:
```suggestion
          <p className="text-4 leading-[19px] font-medium not-italic">Your site</p>
```

File: src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+          <input
+            className="box-border pl-[3px] w-[280px] h-[44px] mb-[24px] border-solid border-[1px] border-[#03111B] rounded-[5px]
```
Comment (smarthulala) [inline src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx:19]: border-solid border-[1px] === border

File: src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+        <div>
+          <p className="text-[16px] leading-[19px] font-normal not-italic text-[#1E1E1E66]">
+            By clicking below, you agree to the Crankbit Cloud Terms of Service and Privacy Policy.
+          </p>
+        </div>
+      </div>
+
+      <button
+        type="button"
+        className="w-[280px] h-[50px] text-[#FFFFFF] bg-[#007AD3] text-[18px] leading-[22px] font-bold rounded-[5px] mt-5"
```
Comment (smarthulala) [inline src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx:33]: customised tailwind pixels that is a multiple of 4, there is a default tailwind value
for example: 20px === 5, 16px === 4

**Review by smarthulala — CHANGES_REQUESTED** [review]

**Comment by Ry4n6137** [comment]
> What's the router url for this page?

I've added it

**Comment by Ry4n6137** [comment]
I‘m waiting for rereview

File: src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+  <div className="flex justify-center items-center bg-white w-[640px] h-[832px]  ">
+    <div className="relative text-left w-[280px] h-[356px]">
+      <h1 className="text-[#1E1E1E] text-4.5 mb-5 font-medium">Welcome back, Alonso</h1>
+      <div>
+        <div>
+          <p className="text-4 leading-5 font-medium not-italic">Work email</p>
+          <input
+            className="box-border pl-[3px] w-[280px] h-11 mb-6 border border-[#03111B] rounded-[5px]
+             placeholder-shown:border-gray-500"
+            placeholder="  Alonso@mail.com"
```
Comment (Qiming-Liu) [inline src/pages/Login/components/LoginForCompany/LoginPage/LoginPage.tsx:13]: Should we remove this space?

**Review by Qiming-Liu — CHANGES_REQUESTED** [review]

**Pull request merged**